tests/unit/cli/test_types.py

```diff
@@ -241,6 +241,64 @@ def test_successful_parsing(self, input, expected):
         self.assertEqual(result, expected, msg="Failed with Input = " + str(input))
 
 
+class TestCfnTagsMultipleValues(TestCase):
+    """
+    Tests for the CfnTags parameter allowing multiple values per key.
+    """
+
+    def setUp(self):
+        self.param_type = CfnTags(multiple_values_per_key=True)
+
+    @parameterized.expand(
+        [
+            # Just a string
+            ("some string"),
+            # Wrong notation
+            # ("a==b"),
+            # Wrong multi-key notation
+            # ("a==b,c==d"),
```

Comment:
Why commented?

Reply:
The comments were in the original `CfnTags` tests, I'm not entirely sure why